src/goose/profile.py

```diff
@@ -49,6 +49,6 @@ def default_profile(provider: str, processor: str, accelerator: str, **kwargs: D
         provider=provider,
         processor=processor,
         accelerator=accelerator,
-        moderator="truncate",
+        moderator="summarize",
```

Comment:
We need to get the token offset / fraction of token count into the Moderator to enable this by default. Currently, it'll summarize the entire context and replace, which I don't think we want.

Reply:
Yup!